examples/Hermitian_Phasing.py

```diff
@@ -3,40 +3,39 @@
 ============================================
 
 Take fake data with a relatively symmetric echo 
-(:math:`T_2^*=1/50\pi`, echo time of 10 ms),
+(:math:r`T_2^*=1/50 pi`, echo time of 10 ms),
```

Comment:
Stopping here. Previous comments not resolved

Reply:
I honestly don't understand why this file is being included in this PR

Reply:
permission to remove from PR?

Reply:
no, someone did work to make this file OK for flake, and I've reviewed the rest of it just fix this problem.  This is a silly discussion